### PR TITLE
SWAT_viz scripts ready to merge

### PR DESCRIPTION
**Pull request by erarsena**
I have 2 scripts here ready for your review. I hope I am doing this right!

### CODE REVIEW

File: SWAT_viz/collatedPlots.R [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+folder1_id = drive_ls(path = as_id(did), pattern = 'SWAT_collatedFiles')$id
+
+folder2_id = drive_ls(path = as_id(folder1_id), pattern = 'collated')$id #get folder id
+
+csv_files <- drive_ls(folder2_id, type = "csv")
+
+#downloads all files in the "collated" folder into temporary folder in R
+csv_files %>% 
+  split(csv_files$id) %>%
```
Comment (steeleb) [inline SWAT_viz/collatedPlots.R:32]: This is an awesome use of `walk` - so much more agile than the loop!

File: SWAT_viz/collatedPlots.R [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+csv_files <- drive_ls(folder2_id, type = "csv")
+
+#downloads all files in the "collated" folder into temporary folder in R
+csv_files %>% 
+  split(csv_files$id) %>% 
+  walk(~drive_download(.$id, path = file.path("tmp", .$name), overwrite = TRUE))
+
+#reads in all collated files
+swat_collatedAnnual <- list.files(path="tmp/", pattern = "yr_collate", full.names = T) %>% 
+  lapply(read.csv) %>%
```
Comment (steeleb) [inline SWAT_viz/collatedPlots.R:37]: FWIW, you could use `map` here, which is in the same family as walk (part of the `purrr` package)

**Review by steeleb — APPROVED** [review]
Thanks for scripting these. They look phenomenal to me. Will you pop a README in there the next time you go to update these scripts? I should have done that when I created this subdir but didn't.

File: SWAT_viz/collatedPlots.R (hunk 2 above)
Comment (erarsena) [reply SWAT_viz/collatedPlots.R:37]: ooh cool thanks!

**Comment by erarsena** [comment]
Thanks so much! Yes will add a README!

**Pull request merged**